paths/graphql/04-query-info.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: simple-class
+header:
+  overlay_image: cover.jpeg
+  overlay_filter: rgba(46, 129, 200, 0.6)
+title: Grab a Query's Response
+permalink: /graphql/query-info
+next-page: /graphql/another-query
+facilitator: false
+sidebar:
+  nav: "github-graphql"
+main-content: |
+
+  ![img](../images/gifs/graphql/recent-prs.gif)
+
+  1. Using the response from the query in the last step, fill in this template.
```

Comment:
I understand it would require a bit of scripting, but I am not a fan of having the users copy/paste between the query and the mutation. Let's think about this.